blockchainetl/jobs/exporters/s3_item_exporter.py

```diff
@@ -0,0 +1,87 @@
+import collections
+import os, sys
+import logging
+import boto3
+from blockchainetl.jobs.exporters.composite_item_exporter import CompositeItemExporter
+from blockchainetl.jobs.exporters.converters.composite_item_converter import CompositeItemConverter
+from blockchainetl.file_utils import get_file_handle, close_silently
+from datetime import datetime
+import csv
+import json
+
+from uuid import uuid4
+
+class S3ItemExporter(CompositeItemExporter):
+
+    def __init__(self, filename_mapping=None, bucket=None, converters=(), environment='dev', chain='ethereum'):
```

Comment:
It seems the filename_mapping parameter is not used

Reply:
yes this is redundant, will clean it up